docs/content/installation/migration/ingress-nginx.md

```diff
@@ -0,0 +1,549 @@
+---
+title: "Migrating from Ingress-NGINX Controller to NGINX Ingress Controller"
+date: 2023-09-29T16:31:21+01:00
+description: "This document describes how to migrate from the community-maintained Ingress-NGINX Controller to the F5 NGINX Ingress Controller."
+weight: 100
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-000"
+categories: ["installation", "platform management"]
+doctypes: ["tutorial"]
+journeys: ["getting started"]
+personas: ["devops"]
+authors: ["Jason Williams"]
+---
+
+<br>
+
+## Overview
+
+This page explains two different ways to migrate from the community-maintained [Ingress-NGINX Controller](https://github.com/kubernetes/ingress-nginx) project to the NGINX Ingress Controller: using NGINX's Ingress Resources or with Kubernetes's built-in Ingress Resources. This is typically because of implementation differences, and to take advantage of features such as [NGINX Plus integration]({{<relref "overview/nginx-plus">}}).
```

Comment:
```suggestion
This page explains two different ways to migrate from the community-maintained [Ingress-NGINX Controller](https://github.com/kubernetes/ingress-nginx) project to NGINX Ingress Controller: using NGINX's Ingress Resources or with Kubernetes's built-in Ingress Resources. This is typically because of implementation differences, and to take advantage of features such as [NGINX Plus integration]({{<relref "overview/nginx-plus">}}).
```

Reply:
We don't use articles with product names.